app/views/components/_header-notice.html.erb

```diff
@@ -4,7 +4,11 @@
   heading ||= false
   list ||= []
   call_to_action ||= false
-  branding_classes ||= false
+
+  # branding
+  nhs_branding ||= false
+  branding_classes = ''
+  branding_classes += 'app-c-header-notice__branding--nhs' if nhs_branding
```

Comment:
I can see you're planning ahead, but I think for now it's enough to do something simpler, like:

`branding_classes = 'app-c-header-notice__branding--nhs' if nhs_branding`

and remove line 10.